AccessibilitySnapshot/SnapshotTesting/Classes/SnapshotTesting+Accessibility.swift

```diff
@@ -0,0 +1,187 @@
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SnapshotTesting
+
+extension Snapshotting where Value == UIView, Format == UIImage {
+
+    /// Snapshots the current view with colored overlays of each accessibility element it contains, as well as an
+    /// approximation of the description that VoiceOver will read for each element.
+    public static var accessibilityImage: Snapshotting {
+        return .accessibilityImage()
+    }
+
+    /// Snapshots the current view with colored overlays of each accessibility element it contains, as well as an
+    /// approximation of the description that VoiceOver will read for each element.
+    ///
+    /// - parameter showActivationPoints: When to show indicators for elements' accessibility activation points.
+    /// Defaults to showing activation points only when they are different than the default activation point for that
+    /// element.
+    /// - parameter useMonochromeSnapshot: Whether or not the snapshot of the `view` should be monochrome. Using a
+    /// monochrome snapshot makes it more clear where the highlighted elements are, but may make it difficult to
+    /// read certain views. Defaults to `true`.
+    /// - parameter markerColors: The array of colors which will be chosen from when creating the overlays
+    public static func accessibilityImage(
+        showActivationPoints activationPointDisplayMode: ActivationPointDisplayMode = .whenOverridden,
+        useMonochromeSnapshot: Bool = true,
+        markerColors: [UIColor] = []
+    ) -> Snapshotting {
+        guard isRunningInHostApplication else {
+            fatalError("Accessibility snapshot tests cannot be run in a test target without a host application")
+        }
+
+        return Snapshotting<UIView, UIImage>.image.pullback { view in
+            let containerView = AccessibilitySnapshotView(
+                containedView: view,
+                viewRenderingMode: .renderLayerInContext,
+                markerColors: markerColors,
+                activationPointDisplayMode: activationPointDisplayMode
+            )
+
+            let window = UIWindow(frame: UIScreen.main.bounds)
+            window.makeKeyAndVisible()
+            containerView.center = window.center
+            window.addSubview(containerView)
+
+            containerView.parseAccessibility(useMonochromeSnapshot: useMonochromeSnapshot)
+            containerView.sizeToFit()
+
+            return containerView
+        }
+    }
+
+    /// Snapshots the current view using the specified content size category to test Dynamic Type.
+    ///
+    /// - parameter contentSizeCategory: The content size category to use in the snapshot
+    public static func image(
```

Comment:
Since we don't have any tests for this currently, I'm tempted to leave this out of the public API until we add it across the framework (#4).

@Sherlouk have you been using this functionality?

Reply:
I haven't used this part of the API - we handle dynamic text size differently in our app so do this through a non-trait method. 

Happy to make internal.

Reply:
Addressed this in 5a417e2